src/docs/design/design-audit.mdx

```diff
@@ -0,0 +1,212 @@
+
+## Presentation
+
+Auditors may tune their presentation depending on who they will be presenting to. For example, if the audiences possess a more technical background, consider explaining the styling in CSS notations; if they are more on the product design side, consider showing your raw design process and inviting them for a design critique.
+
+Some team members might be daunted by the auditor’s authority in design; however, they have been with the project way longer than the auditor and likely know some legacy or technical reasons for why things are done in a certain way. Auditors should encourage teams to challenge the design audit result to foster a better discussion.
+
+Auditors need to remember to pause from time to time in their presentations for the audiences to catch up — likely not many people in the audience are used to thinking the way a UI designer thinks. Auditors should also ask one member of the audience to take notes.
+
+## Recommend Next Steps
+
+Before the end of their presentations, auditors should recommend the next steps for teams to act upon. If the auditor has a technical background, they may list a few items as easy-to-fix quick-win items for the developer. If there are major feature suggestions, the auditor may picture how they will fit into the product’s big picture, and what the R&D timeline would look like.
+
+## Improve Brightlayer UI
+
+During the audit, if auditors noticed anything worth componentizing or that the existing Brightlayer UI standard is too strict, please suggest it to us. Many design guidelines in Brightlayer UI today came from real project design audits we did.
+
+# Understand Audit Results
+
+## Prioritize Issues
+
+Teams must prioritize high-severity issues as they are blocking the application’s functions. These issues must be addressed before the next major release. Meanwhile, mid-severity issues can wait until the next major release, but should still be groomed into the backlog. Low-severity issues are typically considered technical debt.
+High-frequency issues — issues that may not be severe but are exposed to users — should also be prioritized.
+
+## Do Not Skip User Testing
+
+A design audit is not meant to replace user testing. While auditors can flag potential issues based on purely their experience in the design industry, they are not the end user. Teams should conduct user testing to drive major changes while using design audit as a supplemental tool.
+
+## Schedule the Next Audit
+
+Typically, one design audit is enough to cover most issues in the current version of the user interface. Product teams may consider reaching out to another designer after the existing issues have been pointed out.
+
+Whenever there are major changes to the user interface, product teams should conduct a new design audit to ensure they are still on track.
+
+# Design Audit Examples
```

Comment:
I would leave this section out (or just say something like 'If you would like to see examples of audits from the past, reach out to us") until we have a better idea how we want to handle 'private' materials

Reply:
what's the concern? There are no business secrets in the URL

Reply:
I'm not an IT expert, but I know they have had concerns in the past about exposing URL structures for where we host files, source code, etc. But the fact that this is already in a commit has exposed it anyway, so 🤷‍♂️  I guess it can stay until they tell us to remove it.

I would update the blurb to say "If you are an Eaton employee, you can view example audit files below (login required).

Reply:
> If outside-Eaton adopters contacted us asking for an example, would we have one to provide?

No... I can fake one but it will take quite some time to do